docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1214,6 +1222,21 @@ In other words the extension will use whatever cluster `kubectl` uses. The same
 
 At the moment no additional options are provided for further customization.
 
+=== Remote Debugging
+
+To remotely debug applications that are running on a cloud environment, we need to deploy the application as described in the previous section and additionally appending the property `quarkus.kubernetes.remote-debug.enabled=true`. This property will automatically configure the Java application to append the java agent configuration (for example: `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005`) and also the service resource to listen the java agent port. 
```

Comment:
`cloud environment` --> `kubernetes environment`
`and additionally appending the property "quarkus.kubernetes.remote-debug.enabled=true"` --> `and add as new property: "quarkus.kubernetes.remote-debug.enabled=true"`
`and also the service resource to listen the java agent port. ` --> `and also the service resource to listen using the java agent port.`

Reply:
updated